src/core/flow.ts

```diff
@@ -315,7 +319,10 @@ export function Flow({
         return state;
       }
     } else if (phaseConfig.next !== undefined) {
```

Comment:
Re: setting `phase` to `null`, I think this will always be `true`, so we can probably collapse everything into the else case.

Reply:
💯